rcgen/tests/openssl.rs

```diff
@@ -540,3 +544,62 @@ fn test_openssl_pkcs1_and_sec1_keys() {
 	let pkcs8_ec_key_der = PrivateKeyDer::try_from(ec_key.private_key_to_pkcs8().unwrap()).unwrap();
 	KeyPair::try_from(&pkcs8_ec_key_der).unwrap();
 }
+
+#[test]
+#[cfg(feature = "x509-parser")]
+fn test_parse_certificate_with_multiple_domain_components() {
```

Comment:
I think this test should be moved to `tests/generic.rs` since it only relies on `x509-parser`. 

While the static test certificate was generated with the `openssl` command line tool, it's not using the Rust `openssl` crate dependency like the other tests in this file.